device/sycl/src/clusterization/clusterization_algorithm.sycl

```diff
@@ -76,42 +91,65 @@ clusterization_algorithm::output_type clusterization_algorithm::operator()(
     m_copy.get().setup(cell_links)->wait();
     vecmem::data::vector_view<unsigned int> cell_links_view(cell_links);
 
+    vecmem::data::vector_view<device::details::index_t> f_backup_view(
+        m_f_backup);
+    vecmem::data::vector_view<device::details::index_t> gf_backup_view(
+        m_gf_backup);
+    vecmem::data::vector_view<unsigned char> adjc_backup_view(m_adjc_backup);
+    vecmem::data::vector_view<device::details::index_t> adjv_backup_view(
+        m_adjv_backup);
```

Comment:
This code is fine, and you may even argue that this is the more readable way to do this. But a slightly more compact way for passing buffers to SYCL lambdas is like:

```c++
vecmem::data::vector_buffer<...> foo{...};
...
   h.parallel_for<...>(..., [..., foo_view = vecmem::get_data(foo), ...] {...});
```

This is more of an advertisement, since we don't really use that formalism all too much in our code right now. But at one point we could make the code slightly more compact like that. :thinking:

Reply:
Done!